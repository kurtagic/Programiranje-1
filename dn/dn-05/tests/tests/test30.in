45
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
8 50
