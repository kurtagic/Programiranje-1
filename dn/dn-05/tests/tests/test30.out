50: 89
51: 89
52: 90
